subsys/net/lib/aws_jobs/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2018 Nordic Semiconductor
```

Comment:
Don't you need a change in `subsys/net/lib/Kconfig` to include this file?

Reply:
Forgot to add top level will add